src/components/PlantingSites/PlantingSiteCreate.tsx

```diff
@@ -1,10 +1,150 @@
+    const plantingSitesLocation = {
+      pathname: APP_PATHS.PLANTING_SITES,
+    };
+    history.push(plantingSitesLocation);
+  };
+
+  const savePlantingSite = async () => {
+    if (!record.name) {
+      setNameError(strings.REQUIRED_FIELD);
+      return;
+    }
+    const response = await postPlantingSite(record);
+
+    if (response.requestSucceeded) {
+      snackbar.toastSuccess(strings.CHANGES_SAVED);
+      goToPlantingSites();
+    } else {
+      snackbar.toastError();
+    }
+  };
+
+  const gridSize = () => {
+    if (isMobile) {
+      return 12;
+    }
+    return 4;
+  };
+
+  return (
+    <TfMain>
+      <Container maxWidth={false}>
+        <Grid container spacing={3}>
+          <Grid item xs={12}>
+            <Link id='back' to={APP_PATHS.PLANTING_SITES} className={classes.back}>
+              <Icon name='caretLeft' className={classes.backIcon} />
+              {strings.PLANTING_SITES}
+            </Link>
+          </Grid>
+          <Grid item xs={12}>
+            <Typography fontSize='24px' fontWeight={600} margin={theme.spacing(1, 0)}>
+              {strings.ADD_PLANTING_SITE}
+            </Typography>
+            <Typography fontSize='14px' fontWeight={400} margin={theme.spacing(0, 0, 3, 0)}>
+              {strings.ADD_PLANTING_SITE_DESCRIPTION}
+            </Typography>
+          </Grid>
+          <PageSnackbar />
+          <Grid item xs={gridSize()}>
+            <TextField
+              id='name'
+              label={strings.NAME}
+              type='text'
+              onChange={onChange}
+              value={record.name}
+              errorText={record.name ? '' : nameError}
+            />
+          </Grid>
+          <Grid item xs={gridSize()}>
+            <TextField
+              id='description'
+              label={strings.DESCRIPTION}
+              type='textarea'
+              onChange={onChange}
+              value={record.description}
+            />
+          </Grid>
+
+          <Grid item xs={12}>
+            <Typography fontSize='20px' fontWeight={600} margin={theme.spacing(3, 0)}>
+              {strings.BOUNDARIES_AND_PLOTS}
+            </Typography>
+          </Grid>
+          <Grid item xs={12}>
+            <Box
+              sx={{
+                border: '1px solid #A9B7B8',
+                maxWidth: '800px',
+                margin: '0 auto',
+                textAlign: 'center',
+                paddingX: 5,
+              }}
+            >
+              <Typography fontSize='20px' fontWeight={600} margin={theme.spacing(3, 0)}>
+                {strings.IMPORT_BOUNDARIES_AND_PLOTS}
+              </Typography>
+              <Typography fontSize='16px' fontWeight={400} padding={theme.spacing(1, 0)}>
+                {strings.IMPORT_BOUNDARIES_AND_PLOTS_DESCRIPTION}
+              </Typography>
+              <Box sx={{ paddingY: 2 }}>
+                <Button label={strings.CONTACT_US} onClick={() => true} size='medium' />
```

Comment:
for now maybe we can do the same thing the contact us card does when we click?
i understand there's a jira out for allison on what this should do.

Reply:
Oh, yes, I can add that for now!